src/odbc/sql2tds.c

```diff
@@ -149,6 +149,129 @@ _odbc_blob_free(TDSCOLUMN *col)
         TDS_ZERO_FREE(col->column_data);
 }
 
+static TDS_INT
+odbc_convert_table(TDS_STMT * stmt, SQLTVP * src, TDS_TVP * dest, SQLLEN num_rows)
```

Comment:
Handle table conversion from ODBC to TDS format. We first allocate a dummy row to store the metadata of the table, then, for each row, we create a `TDSPARAMINFO` buffer (or parameter list), and then populate the row with its entries.